frontend/packages/dev-console/src/utils/pipeline-utils.ts

```diff
@@ -17,12 +25,17 @@ import {
   PipelineTaskRef,
 } from './pipeline-augment';
 import { pipelineFilterReducer, pipelineRunStatus } from './pipeline-filter-reducer';
+import { PIPELINE_SERVICE_ACCOUNT } from '../components/pipelines/const';
 
 interface Resources {
   inputs?: Resource[];
   outputs?: Resource[];
 }
 
+type ServiceAccountKind = {
```

Comment:
Yes, I saw this. I am not using that because of the secrets type defined there. In our case secrets should be ```{ [name]: string: string }[]``` in the ```ServiceAccount```.